### ISSUE TITLE
How install ?

### ISSUE THREAD

**Issue by castorinop** [fieldOfView/Cura-MeasureTool#3]
How install this ?

**Comment by 5axes** [comment]
create a MeasureTool in the Cura directory and copy the files into this directory ...

**Comment by RobotBobUSA** [comment]
What I did to add this BATA plugin to my Cura Version 4.7.1 was FIRST go to the green Code tab and select to download the zip file from Github, SECOND I wend to the Cura 4.7 program folder on the harddrive where my software is located, in the Cura folder I opened the PLUGIN folder, the created a new file called MeasureTool, the opened that folder and copied the files from the downloaded zip file into that folder. Lastly I shut down and reopened Cura and now when I select on my part there is a new Icon on the left that shows a measuring tape. Seems to work good, but at this time it will not highlight where you click, but it does show coordinates in a window for X,Y, and Z. 
Hope this helps someone else, took me a few try's to figure out how to get this Plugin installed.
